website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,244 @@
+layout: content_layout
+products_used:
+  - Consul
+description: You can configure tokens, policies, and roles to work across multiple datacenters.
+---
+
+You can configure tokens, policies, and roles to work across multiple
+datacenters. This provides central configuration of security policies in the
+primary datacenter and failover for the secondary datacenters. If the primary
+datacenter goes offline, the secondary datacenters will still have the security
+policies in place. 
+
+Note, each datacenter should have ACLs enabled and you need to complete the
+bootstrapping process outlined in the [Securing Consul with ACLs
+guide](/consul/security-networking/production-acls) on the primary datacenter.
+This guide includes the additional ACL replication configuration for the Consul
+agents. 
+
+In this guide, you will setup ACL replication. this is a multi-step process
+that includes:
+
+Setting the `primary_datacenter` parameter on all Consul agents in the primary
+datacenter.  Creating the replication token.  Configuring the
+`primary_datacenter` parameter on all Consul agents in the secondary
+datacenter.  Enabling token replication on the servers in the secondary
+datacenter.  Applying the replication token to all the servers in the secondary
+datacenter. 
+
+This guide is intended to be completed during the initial ACL bootstrapping
+process. 
+
+-> After ACLs are enabled you must have a privileged token to complete any
+operation on either datacenter. The privileged token can be the initial
+`bootstrap` token. 
+
+## Configure the Primary Datacenter
+
+~> Note, if your primary datacenter uses the default `datacenter` name of
+`dc1`, you must set the `datacenter` parameter on the secondary datacenter.
+Otherwise, both datacenters will be named `dc1` and there will be conflicts.
+
+### Consul Servers and Clients
+
+You should explicitly set the  `primary_datacenter` parameter on all servers
+and clients, even though, replication is enabled by default on the primary
+datacenter. Your agent configuration should be similar to the example below.  
+
+```json 
+{ 
+"datacenter": "primary_dc", 
+"primary_datacenter": "primary_dc",
+"acl": { 
+  "enabled": true, 
+  "default_policy": "deny", 
+  "down_policy": "extend-cache",
+  "enable_token_persistence": true 
+  } 
+} 
+```
+
+The `primary_datacenter`
+[parameter](https://www.consul.io/docs/agent/options.html#primary_datacenter)
+sets the primary datacenter to have authority for all ACL information. It is
+important to also set this on the clients, so that the clients can forward API
+requests to the servers. 
+
+Finally, start the agent.
+
+```sh 
+$ consul agent -config-file=server.json 
+```
+
+Complete this process on all agents. 
+
+## Create the Replication Token for ACL Management
+
+Next create the replication token for managing ACLs. The token should be
+created with the following privileges:
+
+acl = "write" which will allow you to replicate tokens.  operator = "read" for
```

Comment:
```suggestion
`acl = "write"` which will allow you to replicate tokens.  `operator = "read"` for
```

Reply:
It might be easier to follow if the sentences of this paragraph were broken up and added as comments to the HCL blob below instead?

Reply:
This was also supposed to be a bulleted list.